Content.Server/GameObjects/Components/Damage/DamageThreshold.cs

```diff
@@ -60,5 +60,24 @@ public DamageThresholdPassedEventArgs(DamageThreshold threshold, bool passed, in
             ExcessDamage = excess;
         }
     }
+
+    public class DamageEventArgs : EventArgs
```

Comment:
I feel like something that lets us specify damage source would be great here.

Reply:
So like, the source of the damage?

Reply:
yes

Reply:
I just added both a Source and SourceMob after following Cakey's indications. Please feel free to review the changes!